proposals/lip-0046.md

```diff
@@ -111,7 +119,7 @@ seedRevealSchema = {
 
 In this section, we describe the properties in the `validatorReveals` array of the seed reveal object:
 
-* `generatorAddress`: The address of the generator of the block. A valid generator address is 20 bytes long.
```

Comment:
I would just remove the lengths of `generatorAddress` and `seedReveal` from the properties description since it is defined in the schema.

Reply:
Good point. Done